packages/v4/src/content/design-guidelines/content/writing-design-documentation.md

```diff
@@ -0,0 +1,132 @@
+---
+id: Writing design guidelines for PatternFly
+section: UX writing
+---
+
+Design guidelines walk through building design patterns with PatternFly. This page focuses on how to write clear and consistent design documentation. To learn how to contribute design guidelines, see our [contribution guidelines](/contribute/design-guidelines/).
+
+## Content architecture
+
+Use headings, hierarchy, formatting, and lists to organize your design guidelines into digestible sections.
+
+### Headings
+
+Use headings to divide your content into topic-specific sections for easy navigation. Effective headings are:
+
+- Structured around users' main goals.
+- Front-loaded with descriptors for easy scanning.
+- Attentive to parallel structure.
+- Written in sentence case.
+
+Headings should cascade levels without skipping––always progress from level to level, H1 to H2 to H3. Put your headings in H2 and subheadings in H3. If more sections appear beneath your subheading, style them in H4.
+
+Example:
+
+![An example of how different headings cascade in design documentation. We removed the body copy between them for easy comparions between levels. It reads: "Menu (H1), Vertical navigation (H2), When to use (H3)."](./img/desguidelines1.png)
+
+## Formatting 
+
+### Capitalization
+Our [capitalization style guidelines](/ux-writing/capitalization/) outline best practices for capitalization across PatternFly. 
+
+Always write headings in sentence case. Write component names in lowercase.
+
+Example: 
+
+"Usage guidelines for card views"
+
+### Bolds and italics
+
+**Bold** componennt names when you first introduce them.
+
+Example: A **card** is a square or rectangle container of related information.
+
+### Lists
+
+Lists package long, complex content into shorter, scannable sections. Use lists to introduce use cases, exceptions, and procedures.
+
+When building lists, follow these best practices:
+
+- Use bullets by default. Only use numbers if the order of items is important, like in a callout list.
+- Introduce your list with a clause or phrase that describes the information it shares. 
+- Start each item with a capital letter.
+- End list items with a period if they're sentences, or if they form a full sentence when combined with the clause/phrase that introduces the list.
+- Maintain parallel structure. Start every list item with the same part of speech. (Each item of this list starts with a verb.)
+
+## Images
+
+Design documentation includes images to contextualize elements in a UI and demonstrate them in action. All images should include alt text for accessibility.
+
+### Annotations
+
+Callouts demonstrate your design in action by highlighting specific areas in an image.
+
+Use numbered callouts to call out multiple areas in your image, then use a corresponding numbered list to explain each one.
+
+Example:
```

Comment:
I would almost suggest indenting this example (the image, and the bullet list underneath it), because the bulleted list initially appears to be part of the guidelines, even though it's actually part of the example!